### PR TITLE
Add "start minimized" feature

### LINKED ISSUE #184: Feature: Start-up to system tray
Title

### PR DESCRIPTION
**Pull request by tdaugaard**
- Add `startMinimized` config settings.

- Add checkbox to "_General Settings_" and make it disabled if `startUp` is not set.

- Add CSS to make text and checkbox look better when disabled.

Fixed #184

### CODE REVIEW

File: src/renderer/App.css [hunk 1]
```diff
@@ -71,6 +71,11 @@ div#root {
   padding: 0;
 }
 
+.MuiFormControlLabel-root .MuiFormControlLabel-label.Mui-disabled,
+.Mui-disabled .MuiCheckbox-root .MuiSvgIcon-root {
+  color: #666 !important;
```
Comment (tdaugaard) [inline src/renderer/App.css:76]: I had to put `!important` on the CSS rule as I could not for the life of me figure out where the styling was set and I couldn't get more specific with the rule I made. Suggestions are welcome.

**Comment by tdaugaard** [comment]
To my knowledge, it's not possible to tell whether the app was launched by Windows or started manually and therefore it will _always_ start minimized if set to start with Windows _and_ to start minimized even if started manually.

The only workaround is to set a flag when Windows starts it like `--autostart` or `--hidden` and look for that argument, but I don't know if you want to go that far.

**Review by aza547 — COMMENTED** [review]
Thanks for taking this, one main comment :) 

I'd like to decouple the "run on startup" and "start minimized" feature such that the app will always start minimized if the checkbox is selected. 

Also changelog entry please :)

File: src/settings/GeneralSettings.tsx [hunk 2]
```diff
@@ -112,6 +112,9 @@ export default function GeneralSettings() {
       <FormGroup sx={formGroupStyle}>
         <FormControlLabel control={getCheckBox("startUp")} label="Run on startup" style = {formControlLabelStyle} />
       </FormGroup>
+      <FormGroup sx={formGroupStyle}>
+        <FormControlLabel control={getCheckBox("startMinimized")} label="Start minimized when starting with Windows" style={formControlLabelStyle} disabled={!config.startUp}/>
```
[the remark this answers is not in the document]
Comment (aza547) [reply src/settings/GeneralSettings.tsx:116]: oh and I think with the decoupling the disabled bit can go away here, as can the CSS changes

**Pull request merged**